From ddfe19535200c19637f96ace845b04317ca4feee Mon Sep 17 00:00:00 2001
From: Olivier Roussel <olivier.roussel@inria.fr>
Date: Thu, 2 Nov 2023 11:43:01 +0100
Subject: [PATCH] [all] Fix headeronly extlibs licenses (#4272)

* add missing license file for nlohmann json.

* homogenize license files names.
---
 extlibs/difflib/{LICENCE => LICENSE.MIT} |  0
 extlibs/json/LICENSE.MIT                 | 21 +++++++++++++++++++++
 extlibs/stb/{LICENCE => LICENSE.MIT}     |  0
 3 files changed, 21 insertions(+)
 rename extlibs/difflib/{LICENCE => LICENSE.MIT} (100%)
 create mode 100644 extlibs/json/LICENSE.MIT
 rename extlibs/stb/{LICENCE => LICENSE.MIT} (100%)

diff --git extlibs/difflib/LICENCE extlibs/difflib/LICENSE.MIT
similarity index 100%
rename from extlibs/difflib/LICENCE
rename to extlibs/difflib/LICENSE.MIT
diff --git extlibs/json/LICENSE.MIT extlibs/json/LICENSE.MIT
new file mode 100644
index 0000000000..1c1f7a690d
--- /dev/null
+++ extlibs/json/LICENSE.MIT
@@ -0,0 +1,21 @@
+MIT License 
+
+Copyright (c) 2013-2022 Niels Lohmann
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
diff --git extlibs/stb/LICENCE extlibs/stb/LICENSE.MIT
similarity index 100%
rename from extlibs/stb/LICENCE
rename to extlibs/stb/LICENSE.MIT
-- 
2.34.1

